src/components/menus/content-contextual-menu.js

```diff
@@ -86,6 +93,103 @@ const ContentContextualMenu = (props) => {
         setOpenDialog(dialogId);
     };
 
+    const duplicateItem = () => {
+        if (activeElement) {
+            const duplicateSuffix = '(1)';
+            elementExists(
+                selectedDirectory.elementUuid,
+                activeElement.elementName + duplicateSuffix,
```

Comment:
Maybe use a local variable, set to : activeElement.elementName + duplicateSuffix
and use it elsewhere